internal/cli/runner_profile_set.go

```diff
@@ -269,7 +278,14 @@ func (c *RunnerProfileSetCommand) Flags() *flag.Sets {
 			Name:    "target-runner-id",
 			Target:  &c.flagTargetRunnerId,
 			Default: "",
-			Usage:   "ID of the remote runner to target for the profile.",
+			Usage:   "ID of the runner to target for this remote runner profile.",
+		})
+
+		f.StringMapVar(&flag.StringMapVar{
+			Name:   "target-runner-label",
+			Target: &c.flagTargetRunnerLabels,
+			Usage: "Labels on the runner to target for this remote runner profile. " +
```

Comment:
Maybe this could use an additional sentence on what labels for targeting is used for?